File: src/main/java/org/example/Java/Act12.java
package org.example.Java;

import java.util.Scanner;

public class Act12 {

    Scanner scanner = new Scanner(System.in);
    public int CadenaAlrevesNum() {
        System.out.println("Numero ?");
        int cadena = scanner.nextInt();
        String cadenaTexto = String.valueOf(cadena);
        String cadenaAlreves = "";


        for (int i = 0; i < cadenaTexto.length(); i++) {
            cadenaAlreves = cadenaTexto.charAt(i)+cadenaAlreves;
        }

        int cadenaAlrevesNum = Integer.parseInt(cadenaAlreves);
        return cadenaAlrevesNum;

    }
}

File: src/main/java/org/example/Kotlin/Act12.kt
package org.example.Kotlin

class Act12 {
    fun CadenaAlrevesNum(): Int {
        println("Numero ?")
        val cadena: Int = readLine()!!.toInt()
        val cadenaTexto = cadena.toString()
        var cadenaAlreves = cadenaTexto.reversed()

        val cadenaAlrevesNum = cadenaAlreves.toInt()
        return cadenaAlrevesNum
    }
}